app/styles/wizard.less

```diff
@@ -70,7 +70,7 @@
                 }
             }
 
-            a { outline: 0; }
+            a { outline: 0; cursor: pointer; }
```

Comment:
By default, anchors should already get `cursor: pointer`. What problem is this trying to solve?